### DISCUSSION TITLE
Mega Menu in Orchard CMS

### DISCUSSION THREAD

**Discussion by tripurarisingh1** [OrchardCMS/OrchardCore#13749] [Q&A]
I wanted to implement Mega menu in Orchard CMS and unable to implement it, Can anyone help me on this ?

**Comment by hishamco** [comment]
Nothing should stop you from implementing it using `OC.Menu`, please refer to https://[docs.orchardcore.net](https://docs.orchardcore.net/en/latest/docs/reference/modules/Menu/)/en/latest/docs/reference/modules/Menu/

**Comment by ns8482e** [comment]
@tripurarisingh1 are you referring to full width drop-down menu? You can set 100% width in your css

**Comment by tripurarisingh1** [comment]
got it thank you.
